test_runner/regress/test_tenant_size.py

```diff
@@ -16,6 +16,7 @@
 from fixtures.types import Lsn, TenantId, TimelineId
 
 
+@pytest.mark.xfail
```

Comment:
I was just running this test, and started wonder why this is marked as xfail. Was this committed by accident back in June?

Reply:
If I remember there was some funny race condition with this where it would nondeterministically fail